src/main/java/org/zalando/nakadi/service/EventTypeService.java

```diff
@@ -269,7 +271,43 @@ public void delete(final String eventTypeName, final Client client)
         }
     }
 
-    private void updateRetentionTime(final String eventTypeName, final Long retentionTime)
+    private void updateEventTypeInDB(final EventType eventType, final Long newRetentionTime,
+                                     final Long oldRetentionTime) throws NakadiException {
+        final NakadiException exception = transactionTemplate.execute(action -> {
+            try {
+                eventTypeRepository.update(eventType);
+                updateTimelinesCleanup(eventType.getName(), newRetentionTime, oldRetentionTime);
```

Comment:
@v-stepanov it is not clear how the event type cache will be updated? Maybe it make sense just to switch it with the previous line.